CHANGELOG.md

```diff
@@ -2,13 +2,16 @@
 
 ## 0.7.9dev
 
+
+* [Feature] Modified `histogram` command to support data with NULL values ([#176](https://github.com/ploomber/jupysql/issues/176))
```

Comment:
No need to add `https://github.com/ploomber/jupysql/issues/176`. this is automatically expanded. please add the issue number only